### DISCUSSION TITLE
Conflicts between nerd fonts and systems fonts (private use area)

### DISCUSSION THREAD

**Discussion by afbjorklund** [ryanoasis/nerd-fonts#1344] [Q&A]
I had some conflicts with some system fonts, for languages that I don't really use. So I simply removed them:

```json
  "cod-error": {
    "char": "",
    "code": "ea87"
  },
```

`sudo apt remove fonts-arphic-ukai fonts-arphic-uming`

But one glyph conflict remains, regarding `\uf42e` which also appears in DejaVu Sans... Is this to be expected ?

```json
  "oct-check": {
    "char": "",
    "code": "f42e"
  },
```

https://dejavu.sourceforge.net/samples/DejaVuSans.pdf

**Comment by Finii** [comment]
Hmm, at least in the DejaVu we use there is no glyph at `F42E`:

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/7e6dcf4b-94b7-48ee-bce2-24a21ec1aa55)

and 2.37 is the latest release, at least at https://github.com/dejavu-fonts/dejavu-fonts

**Later**

Ah `DejaVu Sans` and not `DejaVu Sans Mono`...

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/4426d948-698c-4544-86c5-222f3d691382)

> Is this to be expected?

Well, everyone can put anything into the PUA.

You can patch a Nerd Font
* Overwriting the existing glyphs
* Keeping the existing glyphs just patching into free glyphs
* Patch without the Octicons set (`F400` - `F532`)

If you on the other hand work with fontfallback you can create a fontconfig that prefers the Octicons or the DejaVuSans glyphs - not sure what you need.

**Reply by afbjorklund** [reply]
Changing the terminal font was a good enough workaround, I guess that tweaking fontconfig could have worked too...
